pkg/label/filter.go

```diff
@@ -0,0 +1,96 @@
+package label
```

Comment:
Maybe we should move this to something like `pkg/filters/labels` or `pkg/filters/operations`.

Then we can move the security filters from `pkg/security/filters` to `pkg/filters/security` as well.